README.md

```diff
@@ -1,3 +1,8 @@
+# run in development
+```shell
+php -S 127.0.0.1:8080
```

Comment:
If I run it like that,the css files and bootstrap will not appear ,so I must open it with apache

Reply:
You didn't install the bootstrap property.
You either need to use CDN or add the bootstrap script into your repository because it's missing. 
My PhpStorm says, "Cannot resolve directory 'bootstrap-4.3.1' ". You did not add it to your repository. 
But it is not the problem of my pull request what I am trying to solve. The trouble in your bootstrap is in your repository already exists; you need to solve it separately.

Reply:
I did not push it right,but I think that s not the real problem

Reply:
Btw. Why do you use bootstrap version 4? Not version 5? 
https://getbootstrap.com/docs/5.0/getting-started/introduction/

And it's better to use [CDN](https://www.cloudflare.com/learning/cdn/what-is-a-cdn/) for installing bootstrap. 
Such 
```
<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/js/bootstrap.bundle.min.js" integrity="sha384-b5kHyXgcpbZJO/tY9Ul7kGkf1S0CWuKcCD38l8YkeH8z8QjE0GmW1gYU5S9FOnJ0" crossorigin="anonymous"></script>
```

Reply:
Well I did not pay attention to the version